examples/advanced/main.tf

```diff
@@ -2,13 +2,18 @@ provider "logtail" {
   api_token = var.logtail_api_token
 }
 
+resource "logtail_source_group" "group" {
+  name = "Terraform Advanced Source Group"
+}
+
```

Comment:
Thanks for adding it to the examples 🙌  Could you please bump the version in corresponding `versions.tf`? 🙏

Reply:
Missed that one!